test/xref2/module_type_of_size.t/gen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
As I said in https://github.com/ocaml/odoc/pull/1042, this test is not portable and is annoying to work with due to many files and the scripts. Here's an alternative that should be more portable and is easier to read and change: https://github.com/jonludlam/odoc/pull/43

Reply:
I find your version slightly less readable, and it's also more tricky to try outside of 'dune runtest'.

Reply:
Trying outside of `dune runest` is not a goal of cram tests. Perhaps the test should exist in two different forms.
Why is it less readable ? It contains basically the same code but with 90% of it removed.

Reply:
I prefer my tests to be very explicit and clear, and your version, to me, obfuscates it with the source on one line with shell variable substitution going on. Please can we leave it as it is, and if you have any problems maintaining the test just punt it to me :-D

Reply:
Ok, let's leave it as it is.